scripts/commit_count/plot_commit_count.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+plt.rcParams["figure.figsize"] = [120, 8]
+plt.rcParams["figure.dpi"] = 200
+plt.rcParams["font.size"] = 3
+
+
+import pandas as pd
+
+
+def generate_plot(result_dir: str):
+    commit_count_file = os.path.join(result_dir, 'commit_count.csv')
+
+    if not os.path.exists(commit_count_file):
+        print('Commit count CSV file not found for the {}'.format(result_dir))
+
+        return
+
+    commit_count_content = pd.read_csv(
+        commit_count_file,
+        sep=',',
+        header=0,
+        names=['date', 'commit_count']
+    )
+
+    x_pos = range(len(commit_count_content))
+
+    plt.bar(x_pos, commit_count_content['commit_count'])
+
+    plt.xticks(x_pos, commit_count_content['date'], rotation=90)
+
+    plt.ylabel('Commit Count')
+
+    plt.xlabel('Date')
+
+    plt.title(os.path.split(result_dir)[-1])
+
+    plt.savefig(os.path.join(result_dir, 'commit_count.png'))
```

Comment:
couldn't help but notice your code is very spacious in parts. Lot's of empty lines 😆

Reply:
The result of many frustrating experiments with the `matplotlib`. Will fix it

Reply:
No, wait, that's how I always write Python code. Is it really that unreadable?

Reply:
normally, it's good to group lines of code and use blank lines when starting another group. it's good for readability.